packages/vue-email/src/components/e-heading.vue

```diff
@@ -1,46 +1,66 @@
 <template>
   <component
```

Comment:
Binding, like in the previous case, is important because it allows the user to utilize the native tag props.